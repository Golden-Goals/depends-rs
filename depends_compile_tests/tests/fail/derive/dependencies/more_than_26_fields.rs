use depends::derives::Dependencies;

#[derive(Dependencies)]
struct Bar {
    node1: usize,
    node2: usize,
    node3: usize,
    node4: usize,
    node5: usize,
    node6: usize,
    node7: usize,
    node8: usize,
    node9: usize,
    node10: usize,
    node11: usize,
    node12: usize,
    node13: usize,
    node14: usize,
    node15: usize,
    node16: usize,
    node17: usize,
    node18: usize,
    node19: usize,
    node20: usize,
    node21: usize,
    node22: usize,
    node23: usize,
    node24: usize,
    node25: usize,
    node26: usize,
    node27: usize,
}

fn main() {}
