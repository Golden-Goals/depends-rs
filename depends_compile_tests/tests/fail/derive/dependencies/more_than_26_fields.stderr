error: Dependencies only supports structs with up to 26 fields.
 --> tests/fail/derive/dependencies/more_than_26_fields.rs:3:10
  |
3 | #[derive(Dependencies)]
  |          ^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Dependencies` (in Nightly builds, run with -Z macro-backtrace for more info)
